internal/persistence/persistors/mysqlstore/store_organization.go

```diff
@@ -0,0 +1,241 @@
+package mysqlstore
+
+import (
+	"context"
+	"fmt"
+	"github.com/dembygenesis/local.tools/internal/model"
+	"github.com/dembygenesis/local.tools/internal/persistence"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/assets/mysqlmodel"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/mysqltx"
+	"github.com/dembygenesis/local.tools/internal/sysconsts"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+	"github.com/volatiletech/sqlboiler/v4/queries/qm"
+)
+
+func (m *Repository) DeleteOrganization(
+	ctx context.Context,
+	tx persistence.TransactionHandler,
+	id int,
+) error {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return fmt.Errorf("get ctx exec: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{
+		ID:       id,
+		IsActive: false,
+	}
+	if _, err = entry.Update(ctx, ctxExec, boil.Whitelist("is_active")); err != nil {
```

Comment:
Lets update boil.Whitelist("is_active")); to boil.Whitelist(mysqlModel.Organizations.TableColumns.IsActive));

^^ non-verbatim, but explore the package mysqlmodel, it has helper properties that will help you get the exact column name as a const.

Reply:
For [example](https://github.com/dembygenesis/theoverwatchtools.cli/pull/31/files#diff-c4c68ade6c1cc30f284e274d5b5ce363c102bb3a854edb106cbae03b3f23e85fR55)

Reply:
boil.Whitelist("is_active")) has now been changed to boil.Whitelist(mysqlmodel.OrganizationColumns.IsActive))